### PR TITLE
Soft 113 update can jess gerald

### PR DESCRIPTION
**Pull request by ArshanKhanifar**

### CODE REVIEW

**Comment by ArshanKhanifar** [comment]
@g-jessmuir  @GeraldNDA  please approve, let's merge this bish in.

**Comment by g-jessmuir** [comment]
wait I think you left merge markers in 😥

File: libraries/codegen-tooling/inc/can_transmit.h [hunk 1]
```diff
@@ -39,6 +31,38 @@
...
+
+#define CAN_TRANSMIT_POWER_ON_AUX_SEQUENCE(ack_ptr, sequence_u16) \
+  ({                                                              \
+    CanMessage msg = { 0 };                                       \
+    CAN_PACK_POWER_ON_AUX_SEQUENCE(&msg, (sequence_u16));         \
+    StatusCode status = can_transmit(&msg, (ack_ptr));            \
+    status;                                                       \
+  })
+
 #define CAN_TRANSMIT_GET_AUX_STATUS(ack_ptr, aux_bat_ov_flag_u8, aux_bat_ut_flag_u8) \
```
Comment (GeraldNDA) [inline libraries/codegen-tooling/inc/can_transmit.h]: Looks like this should've been deleted in the merge, maybe?

File: libraries/codegen-tooling/inc/can_unpack.h [hunk 2]
```diff
@@ -13,13 +13,22 @@
...
+
+#define CAN_UNPACK_POWER_ON_MAIN_SEQUENCE(msg_ptr, sequence_u16_ptr)           \
+  can_unpack_impl_u16((msg_ptr), 2, (sequence_u16_ptr), CAN_UNPACK_IMPL_EMPTY, \
+                      CAN_UNPACK_IMPL_EMPTY, CAN_UNPACK_IMPL_EMPTY)
+
+#define CAN_UNPACK_POWER_ON_AUX_SEQUENCE(msg_ptr, sequence_u16_ptr)            \
+  can_unpack_impl_u16((msg_ptr), 2, (sequence_u16_ptr), CAN_UNPACK_IMPL_EMPTY, \
+                      CAN_UNPACK_IMPL_EMPTY, CAN_UNPACK_IMPL_EMPTY)
+
 #define CAN_UNPACK_GET_AUX_STATUS(msg_ptr, aux_bat_ov_flag_u8_ptr, aux_bat_ut_flag_u8_ptr) \
```
Comment (GeraldNDA) [inline libraries/codegen-tooling/inc/can_unpack.h]: This too

**Comment by ArshanKhanifar** [comment]
@GeraldNDA  @g-jessmuir  please approve

**Comment by ArshanKhanifar** [comment]
okay, I removed them, please approve once the build passes

**Comment by ArshanKhanifar** [comment]
@GeraldNDA  can you approve pls

**Pull request merged**